libs/features/texts/src/acore-text/acore-string.component.ts

```diff
@@ -0,0 +1,20 @@
+import { ChangeDetectionStrategy, Component, inject } from '@angular/core';
+import { ReactiveFormsModule } from '@angular/forms';
+import { AcoreString } from '@keira/shared/acore-world-model';
+import { SingleRowEditorComponent } from '@keira/shared/base-abstract-classes';
+import { TranslateModule } from '@ngx-translate/core';
+import { TooltipModule } from 'ngx-bootstrap/tooltip';
+import { AcoreStringHandlerService } from './acore-string-handler.service';
+import { AcoreStringService } from './acore-string.service';
+import { QueryOutputComponent, TopBarComponent } from '@keira/shared/base-editor-components';
+
+@Component({
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  templateUrl: './acore-string.component.html',
+  standalone: true,
+  imports: [TranslateModule, ReactiveFormsModule, TooltipModule, QueryOutputComponent, TopBarComponent],
+})
+export class AcoreStringComponent extends SingleRowEditorComponent<AcoreString> {
+  override readonly editorService = inject(AcoreStringService);
```

Comment:
since Angular 14 we can access `protected` attributes also from the template, so there is very rarely a reason to use `public` and when we don't it's just because it's old code :)

```suggestion
  protected override readonly editorService = inject(AcoreStringService);
```

Reply:
This change causes this error:
TS2415: Class AcoreStringComponent incorrectly extends base class SingleRowEditorComponent<AcoreString>
Property editorService is protected in type AcoreStringComponent but public in type SingleRowEditorComponent<AcoreString

And changing the editor causes a follow up issue. 
IMO; this is a separate problem to be addressed